scripts/helpers/slashes.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+module.exports = function (str) {
+  return process.platform === 'win32' && str.replace(/\\/g, '/')
```

Comment:
The process.platform check seems inconsistent now since in summary.js I don't do the same.